cpp/src/io/csv/csv_reader.cu

```diff
@@ -197,6 +198,84 @@ __device__ void setBit(gdf_valid_type* address, int bit) {
 		return base_padding + num_columns * column_bytes; 
 	}
 }
+/**
+* @brief Removes the first and Last quote in the string
+*/
+string removeQoutes(string str, char quotechar) {
```

Comment:
1. If the field is in quotations, it removes them. Based on the today's discussion, this means that the first character has to be a quotation char.
2. AFAIK it would keep the quotation.
@mjsamoht Currently this logic is consistent with how we handle quotations. Let me know if I should align the behavior with Pandas instead.